src/vs/editor/contrib/suggest/suggestController.ts

```diff
@@ -292,8 +292,10 @@ export class SuggestController implements IEditorContribution {
 	}
 
 	private _alertCompletionItem({ completion: suggestion }: CompletionItem): void {
-		let msg = nls.localize('arai.alert.snippet', "Accepting '{0}' did insert the following text: {1}", suggestion.label, suggestion.insertText);
-		alert(msg);
+		if (isNonEmptyArray(suggestion.additionalTextEdits)) {
```

Comment:
@pawelurbanski I have changed this so that accepting a suggestion only alerts you when extra edits have been made, e.g. when accepting `foo` also adds an import to the top of the file